README.adoc

```diff
@@ -599,6 +599,28 @@ class SomethingTest < Minitest::Test
 end
 ----
 
+=== Skipping Runnable Method [[skipping-runnable-method]]
+
+Prefer `skip` over `return` for skipping runnable methods.
```

Comment:
It might be good to add a little more information about "runnable methods".

```suggestion
Prefer `skip` over `return` for skipping runnable methods that start with `test_`.
```